src/main/java/mate/controller/AddDriverToCarController.java

```diff
@@ -0,0 +1,43 @@
+package mate.controller;
+
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Car;
+import mate.model.Driver;
+import mate.service.CarService;
+import mate.service.DriverService;
+
+@WebServlet(urlPatterns = "/add/driver")
```

Comment:
may be it is not a mistake, but in a checklist we need to set path in url like /drivers/add, /drivers/all and so on
especially here we need to set path like /cars/drivers/add (from a checklist)
same in controllers and jsp files below